src/components/Settings/Notifications/NotificationsWebhook/index.tsx

```diff
@@ -32,6 +32,9 @@ const defaultPayload = {
   '{{extra}}': [],
   '{{request}}': {
     request_id: '{{request_id}}',
+    requestedBy_email: '{{requestedBy_email}}',
+    requestedBy_username: '{{requestedBy_username}}',
+    requestedBy_avatar: '{{requestedBy_avatar}}',
```

Comment:
The only thing is new installs won't see this default unless you update the `jsonPayload` in settings.ts.

Reply:
Ah, okay. I'll update that!  Looks like it hasn't been updated since webhook notifications were first introduced 3 months ago 👀

And existing installs will need to click the `Reset to Default` button as well, but I don't think there's much we can do about that?

Reply:
Yeah, nothing we can do for existing installs, that's fine.